Week3/week3-homework/actions/action.js

```diff
@@ -0,0 +1,104 @@
+const util = require('util');
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'todo_list_app',
+});
+
+
+const execQuery = util.promisify(connection.query.bind(connection));
+connection.connect();
+
+async function showTable(req, res) {
+  const tableName = req.params.table;
+  try {
+    const showTableData = await execQuery(`SELECT * FROM ` + (tableName));
```

Comment:
suppose I call the following endpoint
`http://lcoalhost:3000/users; DROP TABLE users`
What would happen?